### ISSUE TITLE
Training script for finetuning on local language

### ISSUE THREAD

**Issue by Coder1010ayush** [rorizzz/YOLO-Stutter#3]
Can you share the training script with transformer model used and loss function with dataloader class used

**Comment by rorizzz** [comment]
Hi, we just uploaded our training script. You can check it under `yolo-stutter` folder.
